### PR TITLE
[risk=no][DB-290] source ui tweak

### PR DESCRIPTION
**Pull request by mhigbyflowers**
this add error message if no data found + checks if there is data to show

### CODE REVIEW

File: public-ui/src/app/views/ehr-view/ehr-view.component.css [hunk 1]
```diff
@@ -140,6 +140,41 @@ h5.secondary-display {
     font-size:1.3em;
 }
 
+.loading-dots {
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/views/ehr-view/ehr-view.component.css]: if this is the same as above. could this be moved into a common place ?

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts [hunk 2]
```diff
@@ -184,11 +185,12 @@ export class EhrViewComponent implements OnInit, OnDestroy {
     this.resetSelectedGraphs();
     this.graphToShow = g;
     if (this.graphToShow === GraphType.Sources) {
+      this.treeLoading = true;
       this.subscriptions.push(this.api.getCriteriaRolledCounts(r.conceptId)
-        .subscribe({
+      .subscribe({
           next: result => {
             this.treeData = [result.parent];
-            this.treeData['children'] = result.children.items;
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/views/ehr-view/ehr-view.component.ts:191]: where is this being set now ?

Comment (mhigbyflowers) [reply public-ui/src/app/views/ehr-view/ehr-view.component.ts:191]: It was redundant. It didn't need it to get the next level children.

File: public-ui/src/app/views/ehr-view/ehr-view.component.css (hunk 1 above)
Comment (mhigbyflowers) [reply public-ui/src/app/views/ehr-view/ehr-view.component.css]: Yep. I will add it to the loading component.

**Review by SrushtiGangireddy — APPROVED** [review]
Looks good. thanks.

**Pull request merged**